catkin_ws/src/state_estimation/src/sensors.py

```diff
@@ -19,111 +18,131 @@
 
 
```

Comment:
Sensor() has a few getters() (we could also just retrieve the variable instead, but I think getters make it cleaner)